test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTests.java

```diff
@@ -2440,57 +2440,6 @@ static public void testDefaultValueInAssortedRefWithLongAlignmentArray() throws
 		}
 	}
 
-	/*
-	 * Ensure that casting null to invalid Qtype class will throw a NoClassDef
-	 */
-	@Test(priority=1)
-	static public void testCheckCastValueTypeOnInvalidLtype() throws Throwable {
-		try {
-			String fields[] = {"longField:J"};
-			Class valueClass = ValueTypeGenerator.generateValueClass("TestCheckCastOnInvalidLtype", fields);
-			MethodHandle checkCastOnInvalidLtype = lookup.findStatic(valueClass, "testCheckCastOnInvalidLtype", MethodType.methodType(Object.class));
-			checkCastOnInvalidLtype.invoke();
-		} catch (Exception e) {
-			Assert.fail("Should not throw exception");
-		}
-	}
-	
-	/*
-	 * Ensure that casting null to a value type class will throw a null pointer exception
-	 * This test is disabled since the latest spec from
-	 * https://cr.openjdk.org/~dlsmith/jep401/jep401-20230519/specs/types-cleanup-jvms.html
-	 * no longer requires null check on the objectref for null restricted value type class
-	 */
-	@Test(enabled=false, priority=1, expectedExceptions=NullPointerException.class)
```

Comment:
As `checkcast` may still need to check the case when casting null to a null restricted CheckedType, we can keep this test case as well as `testCheckCastValueTypeOnNonNullType()` and `testCheckCastRefClassOnNull()`. 
Probably them can be renamed to `testCheckCastNullRestrictedTypeOnNull()`, `testCheckCastNullRestrictedTypeOnNonNullType()` and `testCheckCastNullableTypeOnNull()`

They can be disabled for now if they are failing.